### PR TITLE
Increase the speed of test suite startup

### PR DESCRIPTION
**Pull request by cachedout**
Tests now start in ~1s instead of ~45s

### CODE REVIEW

**Comment by cachedout** [comment]
cc: @s0undt3ch

File: tests/integration/__init__.py [hunk 1]
```diff
@@ -581,7 +587,7 @@ def get_script_args(self):
         return script_args
 
     def get_check_ports(self):
-        return set([self.config['runtests_conn_check_port']])
+        return set([self.config['id']])
```
Comment (s0undt3ch) [inline tests/integration/__init__.py]: Ahhhh, now I see what you're doing.... :+1:

Comment (cachedout) [inline tests/integration/__init__.py]: Tee-hee. ;]

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by jtand** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Pull request merged**